package.json

```diff
@@ -20,7 +20,7 @@
     "validate": "yarn prettier:check && yarn lint && yarn stylelint:check",
     "fix": "yarn prettier:fix && yarn lint:fix && yarn stylelint:fix",
     "prepare": "husky install",
-    "heroku-postbuild": "yarn workspace client build && cp -a packages/client/build/. packages/server/public/ && yarn workspace server db:setup"
```

Comment:
out of curiosity  why was the last yarn command removed?

Reply:
Should be fine now - I didn't change anything myself, but I did get some issues as a result of installing firebase. There were some vulnerabilities due to outdated packages. I guess this issue is a result of audit fix.